spec/features/admin/variants_spec.rb

```diff
@@ -58,6 +58,47 @@
     v.unit_description.should == 'bar'
   end
 
+  describe "editing on hand and on demand values", js:true do
+    let(:product) { create(:simple_product) }
+    let(:variant) { product.variants.first }
+
+    before do
+      login_to_admin_section
+    end
+
+    it "allows changing the on_hand value" do
+      visit spree.edit_admin_product_variant_path(product, variant)
+
+      page.should have_field "variant_on_hand", with: variant.on_hand
```

Comment:
We should use the new `expect` syntax in all new specs. But since the rest of the file uses `should`: :see_no_evil:

Reply:
ah, yes, sorry, I switched that detector long ago in my brain otherwise I'd go crazy looking at all the shoulds and thinking about expects and not being able to change them to avoid merge conflicts...